specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -1,67 +1,55 @@
 import "@typespec/rest";
+import "@typespec/http";
 import "@typespec/versioning";
 import "@azure-tools/typespec-azure-core";
 
 using TypeSpec.Http;
 using TypeSpec.Rest;
 using TypeSpec.Versioning;
 using Azure.Core;
+using Azure.Core.Traits;
 
+@doc("The language service conversations API is a suite of natural language processing (NLP) skills that can be used to analyze structured conversations (textual or spoken). The synchronous API in this suite accepts a request and mediates among multiple language projects, such as LUIS Generally Available, Question Answering, Conversational Language Understanding, and then calls the best candidate service to handle the request. At last, it returns a response with the candidate service's response as a payload.\n\n In some cases, this API needs to forward requests and responses between the caller and an upstream service. The asynchronous APIs in this suite enable tasks like Conversation Summarization and Conversational PII detection.")
 @service({
-  title: "Contoso Widget Manager",
+  title: "Microsoft Cognitive Language Service - Analyze Conversations",
+  version: "2022-05-01",
```

Comment:
Let's base this on "2023-04-01": https://github.com/Azure/azure-rest-api-specs/blob/1e71ad94aeb8843559d59d863c895770560d7c93/specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzeconversations.json#L6

Or is the idea to base it on the first GA of the new service and add in the 2023-04-01 added features later?

Reply:
The idea was to start with the 2022-05-01 and then add the additional features from 2023-04-01 later.